policy/modules/services/dbus.if

```diff
@@ -594,6 +594,24 @@ interface(`dbus_watch_system_bus_runtime_dirs',`
 	allow $1 system_dbusd_runtime_t:dir watch;
 ')
 
+########################################
+## <summary>
+##	Read system bus runtime directories.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`dbus_read_system_bus_runtime_dirs',`
+	gen_require(`
+		type system_dbusd_runtime_t;
+	')
+
+	allow $1 system_dbusd_runtime_t:dir read;
```

Comment:
We use the verb `list` instead of `read` for read on directories. Please change the interface name to `dbus_list_system_bus_runtime` and use `list_dir_perms` instead of `read`.

Reply:
I renamed dbus_read_system_bus_runtime_dirs to dbus_list_system_bus_runtime and used list_dir_perms.
Thanks!